packages/cli/src/commands/generate-batch/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Command } from '@commander-js/extra-typings'
+import generateBatchAction from './action.ts'
+
+export const generateBatch = new Command('generate-batch').alias('gb').description(
+  'Generate snark artifacts for a list of circom circuits',
+)
+  .argument(
+    '<optionsPath>',
+    'Path to the options definition json file: { [circomkitJsonPath]: { circuit:string, params: string[][] }}',
+  )
+  .argument('<destination>', 'Destination directory for the generated artifacts')
```

Comment:
Honestly I have difficulty finding what is the best DX:
using `argument` or `options`.
I don't have strong preferences.

I think I haven;t been consistent with the other commands

Reply:
It depends on the action imo. Argument if it's an actual command with its own options, and option if it's just a value you pass to change the behavior of a command.

An exception is when you have 1 command that needs a path, that could be an argument.